From c9230ce05d9596df22443d564a03d8de63f137d6 Mon Sep 17 00:00:00 2001
From: Mat Booth <mat.booth@gmail.com>
Date: Mon, 13 Sep 2021 15:44:32 +0100
Subject: [PATCH] Avoid requiring pcsc development files at runtime

JDK should try to use the versioned name of the .so instead of the
unversioned name
---
 .../unix/classes/sun/security/smartcardio/PlatformPCSC.java   | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/java.smartcardio/unix/classes/sun/security/smartcardio/PlatformPCSC.java b/src/java.smartcardio/unix/classes/sun/security/smartcardio/PlatformPCSC.java
index 38fa3da9e..12086acac 100644
--- a/src/java.smartcardio/unix/classes/sun/security/smartcardio/PlatformPCSC.java
+++ b/src/java.smartcardio/unix/classes/sun/security/smartcardio/PlatformPCSC.java
@@ -46,8 +46,8 @@ class PlatformPCSC {
 
     private static final String PROP_NAME = "sun.security.smartcardio.library";
 
-    private static final String LIB1 = "/usr/$LIBISA/libpcsclite.so";
-    private static final String LIB2 = "/usr/local/$LIBISA/libpcsclite.so";
+    private static final String LIB1 = "/usr/$LIBISA/libpcsclite.so.1";
+    private static final String LIB2 = "/usr/local/$LIBISA/libpcsclite.so.1";
     private static final String PCSC_FRAMEWORK = "/System/Library/Frameworks/PCSC.framework/Versions/Current/PCSC";
 
     PlatformPCSC() {
-- 
2.31.1

